ssbh_data/src/meshex_data.rs

```diff
@@ -260,17 +262,26 @@ mod tests {
     fn convert_meshex_data() {
         let meshex = MeshEx {
             all_data: Ptr64::new(AllData {
-                bounding_sphere: Vector4::ZERO,
+                bounding_sphere: BoundingSphere {
+                    center: Vector3::ZERO,
+                    radius: 0.0
+                },
                 name: Ptr64::new("All".into()),
             }),
             mesh_object_groups: Ptr64::new(vec![
                 MeshObjectGroup {
-                    bounding_sphere: Vector4::new(1.0, 1.0, 1.0, 1.0),
+                    bounding_sphere: bounding_sphere: BoundingSphere {
```

Comment:
There should be a single `bounding_sphere:` here.

Reply:
Whoops, I don't know how I missed that. This has been resolved with the latest commit.